addon/mixins/route.js

```diff
@@ -115,7 +125,15 @@ export default Ember.Mixin.create({
   */
   infinityModel(modelName, options) {
 
-    if (Ember.isEmpty(this.store) || Ember.isEmpty(this.store.find)){
+    if (emberDataVersionIs('greaterThan', '1.0.0-beta.19.2') && emberDataVersionIs('lessThan', '1.13.4')) {
```

Comment:
There was a regression with how it was unpacking `meta` from the payload in those versions, causing Ember Infinity to think all of the records were loaded immediately.